src/Adapter/MSTest.TestAdapter/Helpers/MethodRunner.cs

```diff
@@ -12,6 +12,72 @@ namespace Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter.Helpers;
 
 internal static class MethodRunner
 {
+    internal static TestFailedException? RunWithTimeout(
```

Comment:
Why don't reuse the below one and simply pass a `CancellationToken.None`? Should it work?

Reply:
the parameter type below is CancellationTokenSource which does not have none property.